Library/OcBootManagementLib/OcBootManagementLib.c

```diff
@@ -455,3 +479,50 @@ OcRunFirmwareApplication (
 
   return Status;
 }
+
+EFI_STATUS
+OcLaunchAppleBootPicker (
+  VOID
+  )
+{
+  EFI_STATUS                              Status;
+  APPLE_FIRMWARE_USER_INTERFACE_PROTOCOL  *FirmwareUI;
+  UINT8                                   *aGopAlreadyConnected;
+
+  Status = gBS->LocateProtocol (
+                  &gAppleFirmwareUserInterfaceProtocolGuid,
+                  NULL,
+                  (VOID **)&FirmwareUI
+                  );
+
+  if (EFI_ERROR (Status)) {
+    DEBUG ((DEBUG_INFO, "OC: Cannot locate FirmwareUI protocol - %r\n", Status));
+  } else if (FirmwareUI->Revision != APPLE_FIRMWARE_USER_INTERFACE_PROTOCOL_REVISION) {
+    DEBUG ((
+      DEBUG_INFO,
+      "OC: Launch Apple picker incompatible FirmwareUI protocol revision %u != %u\n",
+      FirmwareUI->Revision,
+      APPLE_FIRMWARE_USER_INTERFACE_PROTOCOL_REVISION
+      ));
+    Status = EFI_UNSUPPORTED;
+  }
+
+  if (!EFI_ERROR (Status)) {
+    //
+    // Location of relevant byte variable within loaded driver.
+    //
+    aGopAlreadyConnected = (VOID *)((UINT8 *)FirmwareUI + sizeof (APPLE_FIRMWARE_USER_INTERFACE_PROTOCOL));
```

Comment:
Modifying this byte flag is definitely valid for the specific 144.0.0.0.0 version of the firmware I was testing, but I guess it could be more robust (e.g. the same version of the protocol functionally doesn't have to have the same memory layout) to write a small patch to the `ConnectGop` method which this affects.